airflow/providers/airbyte/operators/airbyte.py

```diff
@@ -144,3 +147,30 @@ def on_kill(self):
         if self.job_id:
             self.log.info("on_kill: cancel the airbyte Job %s", self.job_id)
             hook.cancel_job(self.job_id)
+
+    def get_openlineage_facets_on_start(self):
+        hook = AirbyteHook(
+            airbyte_conn_id=self.airbyte_conn_id, api_version=self.api_version, api_type=self.api_type
+        )
+
+        response = hook.get_airbyte_connection_info(self.connection_id)
+        if response.status_code != 200:
+            raise AirflowException(f"Failed to get connection info: {response.text}")
```

Comment:
General note: we should NOT raise any errors within the OL methods. If there is an error that makes it impossible to provide OL metadata, all we should do is log an error and return empty OpenLineage object.